packages/shorebird_cli/lib/src/shorebird_web_console.dart

```diff
@@ -0,0 +1,14 @@
+class ShorebirdWebConsole {
+  static Uri buildLink(String path) {
+    return Uri.parse('https://console.shorebird.dev/$path');
+  }
+
+  static Uri buildAppReleaseLink(
```

Comment:
```suggestion
  static Uri appReleaseUri(
```

Same here re: no "build"